catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitQueryRequestImpl.java

```diff
@@ -190,23 +190,23 @@ public State complete_XML_results(org.omg.CORBA.StringHolder results)
   }
 
   @Override
-  public RequestDescription get_request_description() throws ProcessingFault, SystemFault {
```

Comment:
❗️ You can't rename these methods. They override interface methods in generated CORBA api code.